values.schema.json

```diff
@@ -0,0 +1,479 @@
+                    "properties": {
+                        "enable": {
+                            "type": "boolean",
+                            "default": "false"
+                        },
+                        "hostSecretName": {
+                            "type": "string"
+                        },
+                        "devurlsHostSecretName": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "service": {
+                    "additionalProperties": false,
+                    "type": "object",
+                    "properties": {
+                        "annotations": {
+                            "type": "object"
+                        }
+                    }
+                }
+            }
+        },
+        "devurls": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "host": {
+                    "type": "string"
+                }
+            }
+        },
+        "postgres": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "useDefault": {
+                    "description": "Deploys an internal Postgres instance alongside the platform. It is not recommended to run the internal Postgres instance in production. If true, all other values are ignored.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "host": {
+                    "description": "The host of the external postgres instance.",
+                    "type": "string"
+                },
+                "port": {
+                    "description": "The port of the external postgres instance.",
+                    "type": [
+                        "string",
+                        "number"
+                    ]
+                },
+                "user": {
+                    "description": "The user of the external postgres instance.",
+                    "type": "string"
+                },
+                "database": {
+                    "description": "The name of the database that coder will use. It must exist before Coder is installed.",
+                    "type": "string"
+                },
+                "passwordSecret": {
+                    "type": "string"
+                },
+                "sslMode": {
+                    "type": "string",
+                    "enum": [
+                        "disable",
+                        "allow",
+                        "prefer",
+                        "require",
+                        "verify-ca",
+                        "verify-full"
+                    ],
+                    "default": "require"
+                }
+            }
+        },
+        "imagePullPolicy": {
+            "type": "string",
+            "default": "Always"
```

Comment:
Should fill in the enum for this, I think it's just `Always` `IfNotPresent` and `Never`.